docs/_static/style.css

```diff
@@ -0,0 +1,4 @@
+/* Fix display of nbgallery with Sphinx default css */
+.figure .caption .caption-text a.reference.internal {
+    border: 0;
+}
```

Comment:
Actually, I can't convince myself that this line is still relevant for us.  I added it deliberately for a reason in https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/pull/31/commits/e250cf89bf51cebb31f90bbb9d0ba8469fa1c05b, but I can't remember _why_.  It's not really hurting anything; we can always re-evaluate later.